packages/website/components/SidebarLink.tsx

```diff
@@ -100,14 +101,30 @@ export function SidebarLink({
   href,
   isExternal = false,
   isActive = false,
+  isIndependent = false,
   paddingLeft = 'spacingXl',
 }: SidebarLinkProps) {
   const titleStyles = getSectionTitleStyles(isActive, paddingLeft);
   const linksProps = isExternal
     ? { target: '_blank', rel: 'noopener noreferrer' }
     : {};
 
-  return (
+  return isIndependent ? (
+    <Link href={href} passHref>
+      {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+      <a
+        className={cx([
+          styles.link,
+          titleStyles.clickable,
+          titleStyles.sidebarItem,
+        ])}
+        {...linksProps}
+      >
+        {children}
+        {isExternal ? <ExternalLinkIcon variant="muted" /> : null}
+      </a>
+    </Link>
+  ) : (
```

Comment:
Why is this code necessary?

This link should work the same way that external links in the "Introduction" sidebar work

I don't think any change to SidebarLink is necessary, you should only need to add something like this to the Sidebar at line 140

```jsx
<SidebarSection
  links={[{
    title: 'Submit an idea in GitHub',
    slug:
      'https://github.com/contentful/forma-36/discussions/new',
    type: 'link',
  }]} 
  currentPage={currentPage} 
/>
```

This is all you need to do and it will have the same styles and everything

Reply:
On the same matter, I think the copy of this link should be something shorter, look what happens in my screen (I have a mouse and Chrome shows a scrollbar in our Sidebar)

![Screenshot 2022-01-25 at 16 52 29](https://user-images.githubusercontent.com/6597467/151012093-1675466c-c11f-4d3e-95ce-668cfddac2e8.png)

Reply:
ok, I thought we might not necessarily want to render it as list element, but ok, sure, on my way to fix it.